protocols/4b4a80-pcr_enrichment/4b4a80-pcr_enrichment.ot2.apiv2.py

```diff
@@ -0,0 +1,195 @@
+    E6177S/L (for 1-24 DNA samples): Step 5: PCR Enrichment''',
+    'author': 'Steve Plonk <protocols@opentrons.com>',
+    'apiLevel': '2.9'
+}
+
+
+def run(ctx):
+
+    # bring in constant values from json string above
+    [pcr_labware, pcr_cycles, sample_count, post_size_selection_volume,
+     q5_volume, q5_well, i5_volume, i5_well, i7_volume, i7_well,
+     empty_vial_well, spri_beads_volume, spri_beads_well, etoh_volume,
+     etoh_well, te_volume, te_well
+     ] = get_values(  # noqa: F821
+      'pcr_labware', 'pcr_cycles', 'sample_count',
+      'post_size_selection_volume', 'q5_volume', 'q5_well', 'i5_volume',
+      'i5_well', 'i7_volume', 'i7_well', 'empty_vial_well',
+      'spri_beads_volume', 'spri_beads_well', 'etoh_volume', 'etoh_well',
+      'te_volume', 'te_well')
+
+    ctx.set_rail_lights(True)
+    ctx.delay(seconds=10)
+
+    if sample_count < 1 or sample_count > 24:
+        raise Exception('Invalid number of samples (must be 1-24).')
+
+    # turn off rail lights to bring the pause to the user's attention
+    ctx.set_rail_lights(False)
+    ctx.pause(f"""Please pre-cool the
+    temperature module to 4 degrees via settings in the Opentrons app prior to
+    running this protocol. Please transfer Q5 master mix to well {q5_well}, i5
+    to well {i5_well}, i7 to well {i7_well} of the 4 degree temperature module
+    on the OT-2 deck.""")
+    ctx.set_rail_lights(True)
+
+    # setup p300 single channel, p300 multi channel, tips
+    tips300s = [ctx.load_labware("opentrons_96_tiprack_300ul", '3')]
+    tips300m = [
+     ctx.load_labware("opentrons_96_tiprack_300ul", '6'),
+     ctx.load_labware("opentrons_96_tiprack_300ul", '9')]
+    [p300s, p300m] = [
+     ctx.load_instrument(pipette, side, tip_racks=tips)
+     for pipette, side, tips in zip(
+      ["p300_single_gen2", "p300_multi_gen2"],
+      ['left', 'right'], [tips300s, tips300m])]
+
+    # setup temperature module at 4 degrees for Q5 master mix, i5, i7
+    temp = ctx.load_module('Temperature Module', '2')
+    temp_reagents = temp.load_labware(
+        'opentrons_24_aluminumblock_nest_0.5ml_screwcap',
+        'Opentrons 24-Well Aluminum Block')
+    temp.set_temperature(4)
+
+    # setup thermocycler
+    tc = ctx.load_module('thermocycler')
+    tc.open_lid()
+    tc_plate = tc.load_labware(pcr_labware)
+
+    # setup magnetic module (magnets disengaged)
+    mag = ctx.load_module('magnetic module gen2', '4')
+    mag_plate = mag.load_labware(pcr_labware)
+    mag.disengage()
+
+    # sample locations (first three columns of thermocycler plate)
+    post_size_selection_sample = [
+     well for column in tc_plate.columns() for well in column][:sample_count]
+
+    # reagent setup: Q5 mastermix, i5, i7, etoh, te
+    q5, i5, i7, mixture = [
+     temp_reagents.wells_by_name()[well] for well in [
+      q5_well, i5_well, i7_well, empty_vial_well]]
+
+    p300s.pick_up_tip()
+    p300s.mix(10, 250, q5.bottom())
```

Comment:
it's my understanding that `.bottom()` defaults to the actual bottom of the well, so maybe use `.bottom(1)`